### PR TITLE
Added policy document field in the bucket details

### PR DESCRIPTION
**Pull request by sash-21**

### CODE REVIEW

**Review by ganeshjagdale9545 — APPROVED** [review]
Looks good to me

File: cartography/intel/aws/s3.py [hunk 1]
```diff
@@ -257,26 +257,52 @@ def _load_s3_acls(neo4j_session: neo4j.Session, acls: Dict, aws_account_id: str,
...
+    Fetch policy documents for each S3 bucket in the policies list and prepare a list with the necessary details.
+    """
+    bucket_policies = []
+    for policy in policies:
+        bucket_name = policy.get('bucket')
+        client = boto3_session.client('s3')
+        response = get_policy({'Name': bucket_name}, client)
+        policy_document = response.get('Policy', {}) if response else {}
+
+        bucket_policies.append({
```
Comment (ganeshjagdale9545) [inline cartography/intel/aws/s3.py:296]: instead of creating new object can you add `policy_document` key direct to policy object

**Review by ganeshjagdale9545 — CHANGES_REQUESTED** [review]
one minor change needed

File: cartography/intel/aws/s3.py (hunk 1 above)
Comment (sash-21) [reply cartography/intel/aws/s3.py]: Changes done!

**Review by ganeshjagdale9545 — APPROVED** [review]
Looks good to me

**Pull request merged**